docs/development/code_guide.rst

```diff
@@ -5,14 +5,61 @@ Code Development Guidelines
 This document describes the coding requirements and guidelines to be
 followed during the development of PlasmaPy and affiliated packages.
 
-Dependencies
-============
+Code written for PlasmaPy must be compatible with Python 3.6 and
+later. Python 2 is not supported by PlasmaPy.
+
+PlasmaPy requires 
+
+* Astropy 2.0 or later
+* NumPy 1.13 or later
+* scipy 0.19 or later
+* matplotlib 2.0 or later
+
+Setting up an environment for development
+=========================================
+
+To obtain PlasmaPy's source code, use the usual
+
+.. code-block:: bash
+
+  git clone https://github.com/PlasmaPy/PlasmaPy.git 
+
+To be able to import PlasmaPy from your source version:
+
+.. code-block:: bash
+
+  cd PlasmaPy
+  python setup.py develop
+
+If you are not working within a virtual environment, this may end in a
+permission error. Setup procedures for the two most popular virtual
+environments, conda and virtualenv, are listed below.
+
+Conda
+-----
+
+To set up a development environment for PlasmaPy, `the Anaconda
+distribution <https://www.anaconda.com/download/>`_ is strongly
+recommended.
+
+After installing Anaconda, launching any conda environment and entering
+the PlasmaPy's repository root directory, execute the following:
+
+.. code-block:: bash
+
+    conda create -n plasmapy-dev python=3 --file requirements/base.txt
```

Comment:
When I try to run this on my computer I get the following error:
`CondaValueError: could not parse 'numpy (>= 1.13)' in: requirements/base.txt`

Might be worth testing this out on a couple different operating systems or at least noting the ones this has been tested on.

Reply:
interestingly enough, my computer was perfectly happy with `pip install -r requirements/base.txt` from inside virtualenv.

Reply:
I finally got to it and I found a solution - replacing the call to requirements.txt to one to environment.yml via conda env create. It should work. I'll test this alongside writing Windows instructions in a couple of minutes.